node/api/channels_api_usage.md

```diff
@@ -2056,3 +2058,51 @@ The transaction has the following structure:
   | nonce | integer | settler's nonce |
 
 The amounts are the exact amounts stored in the channel object on-chain.
+
+#### Connection keep alive
+
+WebSocket connection handlers are to identify abrupt network disconnects.
+Thus the WebSocket protocol defines special control frames to be used: sending
+`ping` and `pong`. Clients that are run in a browser have no access nor do
+they have any control over those frames. They are to be handled by browsers
+and this could introduce some undesired incompatibilities.
+
+In order to provide best user experience, we've kept the functionality of the
+node to respond with a `pong` control frame to every `ping` received as well
+as enhanced the State Channel's WebSocket API with the option of sending data
+frames that act like the corresponding control frames. Depending on the
+environment to be ran, the client can use either approach.
+
+If no frames had been received for 1 minute, the node will consider the
```

Comment:
I did not look in details where this 1 minute is set in the code. It seems too detailed here, and shall rather be user configurable, with default mentioned in the user configuration schema.

Reply:
This comes from the `cowboy` lib and it is out of the box feature. Please check section `Keeping the connection alive` in [the docs](https://ninenines.eu/docs/en/cowboy/2.1/guide/ws_handlers/). It could as well be connection specific config but it is not exposed to State Channels' clients. I am not really eager exposing it as it could easily lead to clients leaking WebSocket connections on the node.

Reply:
had been -> have been